Makefile

```diff
@@ -26,3 +26,13 @@ check:
 	# Configure project's revive checks using .revive.toml
 	# See: https://github.com/mgechev/revive
 	@for d in $$(go list ./... | grep -v /vendor/); do revive -config .revive.toml $${d}; done
+
+# Do a quick compile only check of the tests and impliclity the
+# library code as well.
+test-binaries: cephfs.test errutil.test rados.test rbd.test
+test-bins: test-binaries
+
+%.test: % phony_go
```

Comment:
I don't get the `phony_go` part. I would expect it to just work without as well.

Reply:
Because makefile doesn't actually build the binaries (or manage the dependencies), I didn't want the make command skipping the build if pkg.test existed. I researched a couple of ways to force make to always run the rule, and the one that looked the cleanest to me was a phony rule that does nothing.

Reply:
I can add a comment to clarify what phony_go does if you think that would help, or feel free to suggest a different way to force it.
FWIW all this is to make it so I can use the make command in vim and do quick(er) checks. :-)

Reply:
Ah, I see. In that case, can you rename `phony_go` to `force_build` or something? A little comment would be useful, but with a more descriptive name it is not really needed.